databases/Spider.yaml

```diff
@@ -0,0 +1,16 @@
+database_name: spider_examples
```

Comment:
I am not sure about the name. I think Max had a different idea about how to name these so that it would make more sense for someone who does not know what Spider is. Something on the line "ai_assist_examples" as far as I remember